README.md

```diff
@@ -332,6 +332,16 @@ $ make bench
 
 Issues and pull requests welcome!
 
+## contributors
+
+* [Ant Skelton](https://github.com/blowback)
+* [Benjamin Byholm](https://github.com/kkoopa)
+* [Bent Cardan](https://github.com/reqshark)
+* [Michele Comignano](https://github.com/comick)
+* [Nick Desaulniers](https://github.com/nickdesaulniers)
+* [Tim Cameron Ryan](https://github.com/tcr)
+* [Trygve Lie](https://github.com/trygve-lie)
```

Comment:
let's use a github table

Ant Skelton | https://github.com/blowback
:-------------: | :-------------:
<strong>Benjamin Byholm  | <strong>https://github.com/kkoopa</strong>
<strong>Bent Cardan  | <strong>https://github.com/reqshark
<strong>Michele Comignano  | <strong>https://github.com/comick
<strong>Nick Desaulniers  | <strong>https://github.com/nickdesaulniers
<strong>Tim Cameron Ryan  | <strong>https://github.com/tcr
<strong>Trygve Lie | <strong>https://github.com/trygve-lie

but the markdown for that is weird, note how i dont bother closing the `<strong>` tags. 

```md
Ant Skelton | https://github.com/blowback
:-------------: | :-------------:
<strong>Benjamin Byholm  | <strong>https://github.com/kkoopa</strong>
<strong>Bent Cardan  | <strong>https://github.com/reqshark
<strong>Michele Comignano  | <strong>https://github.com/comick
<strong>Nick Desaulniers  | <strong>https://github.com/nickdesaulniers
<strong>Tim Cameron Ryan  | <strong>https://github.com/tcr
<strong>Trygve Lie | <strong>https://github.com/trygve-lie
``` 
also looks like github forces us to have a center aligned top column. so i center aligned the rest but i think left alignment would be better. i guess that's people use html tags for this